standards/technical-approach.md

```diff
@@ -22,18 +40,35 @@ TADs SHOULD be created in the following situations:
+
+A proof of concept should be time-boxed to no longer than one sprint (two weeks). Tech leads and architects SHOULD guide engineers in generating a list of questions that the proof of concept aims to answer or clarify.
+
+A proof of concept MUST NOT go to production. The expectation is that a proof of concept is a rough draft built using mock data and may not include the cleanest or most maintainable code. If possible, it is recommended to implement a solution that _only_ runs locally & in a separate repository with a name that includes `proof of concept` in the title OR in a feature branch that is left unmerged.
 
 ## Process
 
 All TADs MUST follow this process to be approved, at which point JIRA tickets can be created and development work can start.
 
-1. The most senior developer on the feature/task/story is assigned a ticket to write the TAD and the ticket MUST be tagged with `tech-approach`
-2. The assigned developer should collect as much information from the BRD, stakeholders and any other relevant information sources and write the TAD
-3. The TAD MAY be circulated at this point for feedback within a team or stakeholders for further input/feedback
+1. An engineer is assigned a ticket to write the TAD and the ticket MUST be tagged with `tech-approach`
+2. The assigned developer should collect as much information from the BRD, stakeholders and any other relevant information sources and write the TAD. It is recommended to time box the first draft of the TAD-writing phase to one sprint (2 weeks) so as not to lose context or momentum on the problem at hand. If a [proof of concept](#proof-of-concepts) is deemed necessary, then this may extend the TAD-writing phase by an additional sprint.
+3. The TAD MAY be circulated at this point for feedback within a team or stakeholders for further input/feedback. Posting the drafted TAD in a public channel is a good way to start receiving feedback. Scheduling a meeting to review the draft with team members is another option. It is recommended to set clear expectations about how long the TAD will be open for comments and discussion from team members and when that window is considered closed.
 4. When ready for review the Engineering Leadership Team (ELT) should be notified, the TAD uploaded to the shared TAD Google Drive (see below), and the associated JIRA ticket moved to the `Under Review` column
 5. The TAD MUST be reviewed by at least one ELT member with subject area knowledge, and SHOULD be reviewed by two members (see review process below)
-6. If approved in Step 5 the TAD should be sent to Garvita Kapur for final approval. With final approval the associated JIRA ticket is moved to `Done`
-7. If approved specific tickets should be created to execute the work as described in the TAD
+6. If approved, in Step 5 the TAD MUST be sent to the Director of Engineering for final approval. With final approval the associated JIRA ticket is moved to `Done`
```

Comment:
Nit: This comma should be moved to after "in Step 5" or just remove "in Step 5" altogether.

Reply:
Nit: move this line, "With final approval the associated JIRA ticket is moved to `Done`" to the next step.

"7. With final approval the associated JIRA ticket is moved to `Done`. Specific tickets MUST be created to execute the work as described in the TAD. "